docs/kyma/04-02-local-deployment.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Deploy Kyma locally
+type: Installation
+---
+
+This Deployment guide shows you how to quickly deploy Kyma locally on the MacOS, Linux, and Windows platforms. Kyma is deployed locally using a proprietary installer based on a [Kubernetes operator](https://coreos.com/operators/).
+
+**NOTE:** Local deployment using Minikube is deprecated.
+
+>**TIP:** See the [troubleshooting guide](#troubleshooting-overview) for tips.
+
+## Prerequisites
+
+<div tabs name="prerequisites" group="deploy-kyma-locally">
+  <details>
+  <summary label="k3s">
+  k3s
+  </summary>
+
```

Comment:
We just discussed the topic with @majakurcius. To guide the end-user a bit more in figuring out how to get the setup done, we should give some more recommendations.
Could you maybe add tabs per supported OS and write per dependency what version to use, what is the recommended way to install it and where to find the instructions?
For the versions it is enough to say:
- kyma CLI: always use the version corresponding to your kyma installation
- docker: 20.x.x
- K3D: 4.x.x